cli/src/install.sh

```diff
@@ -141,6 +152,7 @@ cleanup_pre_xdg() {
 
 cd "$(dirname "$0")"
 banner
+check_glibc
```

Comment:
We only rely on `glibc` on Linux. We need to guard this so that it doesn't run on macos installs.